web/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx"
+  },
+  "include": [
+    "src"
+  ]
```

Comment:
You can add paths to enable @components @services convension

Reply:
We need to add an external dependency to achieve this. See this

https://plusreturn.com/blog/how-to-configure-a-path-alias-in-a-react-typescript-app-for-cleaner-imports/

Let me know if you know a simpler way. Adding paths did not work and I think it will not work in react. We need a way to update web pack also, which can be done using external dependencies only in react

Reply:
https://medium.com/@danil_chepelev/how-to-setup-path-aliases-with-react-typescript-and-webpack-54e579c189e1